name-lint.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+LINT="$(grep -e "Pycon UK" -e "pycon UK" -e "pyconUK" -e "PyConUK" \
+        --line-number --recursive . \
+        --include "*.html" --include "*.rst")"
+
+SUCCESS=0
+
+if [[ $LINT ]] ;  then
```

Comment:
@ghickman fixed I think. The latest commit works for me with `sh` and `bash` on Ubuntu 15.04 and 12.04